pkg/metrics/metrics.go

```diff
@@ -112,6 +120,18 @@ func (metrics *metrics) ObserveUpdateDurationByType(typ string, duration time.Du
 	metrics.Update.With(prometheus.Labels{"type": typ}).Observe(duration.Seconds())
 }
 
+func (metrics *metrics) ObservePipelineStateDuration(state *api.PipelineStatus, key string) error{
+	if state.StartedAt.IsZero() {
+		return fmt.Errorf("cannot observe StateItem if StartedAt is not set")
+	}
+	duration := time.Now().Sub(state.StartedAt.Time)
```

Comment:
does this (always) respect the time zone? Can `time.Now()` be in another time zone as `state.StartedAt.Time`? What would happen?

Reply:
Timezones only come into play when convert a time object from it's internal representation into something human readable like hour, day, month...